src/runtime/gpu/gpu_cuda_kernel_ops.hpp

```diff
@@ -16,54 +16,48 @@
 
 #pragma once
 
+#include "ngraph/op/abs.hpp"
```

Comment:
`ngraph/ops.hpp` was made for just this sort of occasion.

Reply:
I looked for that header in the ops directory. I guess I picked the wrong place to look.
fixed